### PR TITLE
[CosmosDB] Bug Fixes and Enhancements

### PR DESCRIPTION
**Pull request by MehaKaushik**
## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

File: src/CosmosDB/CosmosDB/ChangeLog.md [hunk 1]
```diff
@@ -19,13 +19,17 @@
 -->
 
 ## Upcoming Release
+* Changes in New-AzCosmosDBAccount and Update-AzCosmosDBAccount
```
Comment (msJinLei) [inline src/CosmosDB/CosmosDB/ChangeLog.md:22]: The ident here looks incorrect

File: src/CosmosDB/CosmosDB/ChangeLog.md [hunk 2]
```diff
@@ -19,13 +19,17 @@
...
+            - Renamed ApiKind value GlobalDocumentDB to Sql.
+            - Added parameter DisableKeyBasedMetadataWriteAccess
 * Introduces cmdlets to update throughput for Sql Database and Container, Cassandra Keyspace and Table, MongoDB Database and Collection, Gremlin Database and Graph and Table.
             - Update-AzCosmosDBSqlContainerThroughput, Update-AzCosmosDBSqlDatabaseThroughput,
               Update-AzCosmosDBMongoDBCollectionThroughput, Update-AzCosmosDBMongoDBDatabaseThroughput,
               Update-AzCosmosDBGremlinGraphThroughput, Update-AzCosmosDBGremlinDatabaseThroughput,
               Update-AzCosmosDBCassandraTableThroughput, Update-AzCosmosDBCassandraKeyspaceThroughput,
               Update-AzCosmosDBTableThroughput
-
+
```
Comment (msJinLei) [inline src/CosmosDB/CosmosDB/ChangeLog.md:32]: Redundant spaces

File: src/CosmosDB/CosmosDB/ChangeLog.md (hunk 1 above)
Comment (MehaKaushik) [reply src/CosmosDB/CosmosDB/ChangeLog.md:22]: Please check if it is better now.

**Pull request merged**